app/lib/common/widgets/room/room_with_profile_card.dart

```diff
@@ -129,9 +131,24 @@ class RoomWithAvatarInfoCard extends StatelessWidget {
         titleTextStyle: titleTextStyle,
         subtitleTextStyle: subtitleTextStyle,
         leadingAndTrailingTextStyle: leadingAndTrailingTextStyle,
-        title: Text(title, overflow: TextOverflow.ellipsis),
-        subtitle: buildSubtitle(context),
-        leading: avatar,
+        title: Row(
```

Comment:
This changes are for providing option to added dynamic `leading` widget. Earlier dynamic `trailing` widget option was there.